extraConfigMicroshift.py

```diff
@@ -126,8 +125,9 @@ def ExtraConfigMicroshift(cc: ClustersConfig, cfg: ExtraConfigArgs, futures: dic
     contents = read_prep_microshift_kubeconfig(acc)
     kubeconfig = write_microshift_kubeconfig(contents, host.LocalHost())
 
-    acc.run("systemctl stop firewalld")
-    acc.run("systemctl disable firewalld")
+    # Enable and configure firewall at IPU
+    firewall = Firewall(acc)
```

Comment:
is this also valid for the Marvell DPU? I think not.

In any case, the code comment is wrong and must not be about IPU.